citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -411,6 +411,11 @@ ${author.lastName}<c:if test="${not empty author.firstName}">, ${author.firstNam
 </table>
 
 <button id="submitForm" class="btn btn-primary" type="submit"><i class="far fa-save"></i> &nbsp;Save</button>
+<a href="<c:url value="/auth/group/${zoteroGroupId}/items/${itemId}" />">
+	<button id="cancelButton" type="button" class="btn btn-default">
```

Comment:
why do you have a button here? if it's just for the styling of the link, just give the link the button classes